bbb-graphql-server/bbb-graphql-schema.md

```
@@ -115,7 +115,8 @@ Permission: Restricted to User Viewing Self-Related Data
 - `chats: Array` [Type chat](#type-chat)
 - `connectionStatus: Object` [Type user_connectionStatus](#type-user_connectionstatus)
 - `guestStatusDetails: Object` [Type user_guest](#type-user_guest)
-- `lastBreakoutRoom: Object` [Type user_breakoutroom](#type-user_breakoutroom)
+- `lastBreakoutRoom: Object` [Type breakoutRoom_user](#type-breakoutRoom_user)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid
Expected: #type-breakoutroom_user; Actual: #type-breakoutRoom_user

(MD051, link-fragments)